### DISCUSSION TITLE
GitHub Rejects Student Pack Applications

### DISCUSSION THREAD

**Discussion by Maslo1Bread** [community/community#144699] [GitHub Education]
I have already sent a dozen applications to receive student status, but I am constantly rejected. I have already tried everything. I sent a photo of my student ID, a certificate of my enrollment in studies, a screenshot of the scholarship enrollment, a screenshot of the fact that I am in a group at the college, a screenshot of the educational portal, and I am constantly rejected. My profile is filled out, payment information is also there, two-step verification is also there. I just do not understand what the problem is, the bot constantly answers me with a template message. (sorry for English and potential mistakes, I'm using google translator)

**Comment by seasonedu** [comment]
apply with your phone + turn location service on
